src/iframe/src/colors.js

```diff
@@ -21,6 +21,20 @@ const triplets = [
   [8, 20, 30]
 ]
 
+let intLookup = [];
```

Comment:
Will this logic be run every time the `colors.js` file is read? Should it be part of a function instead?

Reply:
Also I think this whole file needs a bit of documentation. At the very least, I think we should have an example of inputs and outputs on function headers. But I can do that later.

Reply:
Nah ecmascript modules (what webpack emulates in bundled files) only initialize once and the reference to the module is passed every time it is imported.

This does mean that the lookup will be populated at startup if some other bundle imports the color file, but its not a lot of computation for the benefit in runtime.